EndlessSkyLib.cbp

```diff
@@ -355,12 +357,12 @@
 		<Unit filename="source/Weather.cpp" />
 		<Unit filename="source/Weather.h" />
 		<Unit filename="source/WeightedList.h" />
+		<Unit filename="source/comparators/ByGivenOrder.h" />
+		<Unit filename="source/comparators/ByName.h" />
 		<Unit filename="source/opengl.cpp" />
 		<Unit filename="source/opengl.h" />
 		<Unit filename="source/pi.h" />
 		<Unit filename="source/shift.h" />
-		<Unit filename="source/comparators/ByGivenOrder.h" />
-		<Unit filename="source/comparators/ByName.h" />
```

Comment:
Why did you change this file? seems like nothing really changed...

Reply:
I mean the order
not the file

Reply:
It was automatic by Code::Blocks

Reply:
hmmm... ok, maybe change it back? it is not on topic of this pr

Reply:
leave it like it is